BDArmory/Control/BDAirspeedControl.cs

```diff
@@ -556,6 +556,18 @@ void UpdateSAS(FlightCtrlState s)
             ap.SAS.SetTargetOrientation(throttleLerped > 0 && lerpAttitude ? attitudeLerped : attitude, false);
         }
 
+        public void Stability(bool enable)
+        {
+            if (lockAttitude == enable) return;
+            lockAttitude = enable;
+
+            var ap = vessel.Autopilot;
+            if (ap == null) return;
+
+            vessel.ActionGroups.SetGroup(KSPActionGroup.SAS, enable);
+            ap.SetMode(enable ? VesselAutopilot.AutopilotMode.StabilityAssist : VesselAutopilot.AutopilotMode.Normal);
```

Comment:
Why set `Normal` mode when disabling SAS?
This would be better to either store the previous mode when SAS gets enabled and restore it on disabling SAS, or just always setting it to `StabilityAssist`.